site/gatsby-site/cypress/e2e/integration/pages.cy.js

```diff
@@ -111,5 +111,44 @@ describe('Pages', () => {
         }
       });
     });
+
+    it(`Should not display broken images on page ${path}`, () => {
```

Comment:
Sorry, I wasn't very clear. Please move these tests to the `it` above so we run them all in one visit to reduce loading times (at least until we have more powerful boxes ).

(let's see how flaky this gets)